lib/jsdom/level2/html.js

```diff
@@ -164,11 +166,46 @@ ResourceQueue.prototype = {
   }
 };
 
+function RequestManager(options) {
+  this.pool = options.pool;
```

Comment:
Why does this have pool and agentOptions? It's not actually used by this class...